advanced/am-pattern-2/templates/am-analytics/dashboard/wso2am-pattern-2-am-analytics-dashboard-ingress.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: wso2am-pattern-2-am-analytics-dashboard-ingress
+  namespace : {{ .Release.Namespace }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/ssl-passthrough: "true"
```

Comment:
@Chathuru IMO, we don't need SSL passthrough in this case. Setting the back-end protocol to `HTTPS` would suffice. Applies to other places as well.

Can you please check?

Reply:
Fix in https://github.com/Chathuru/kubernetes-apim/commit/0205f6311ce5b9ff81ebe108aa79ff7a040078e1 commit